gassi/gassi.csr

```diff
@@ -0,0 +1,26 @@
+-----BEGIN CERTIFICATE REQUEST-----
+MIIEVTCCAj0CAQAwEDEOMAwGA1UEAwwFZ2Fzc2kwggIiMA0GCSqGSIb3DQEBAQUA
+A4ICDwAwggIKAoICAQCZdHRSQKEXIpk9yMjzlWBrDYwutE81RZBAKEkeuCXqiRD4
+bcs+FnukxFJA1HwDNxMxVst9gxp6ksx/qFDBrdsz3f0S4TqPnsMYr0BjEYdcO0BB
+t5r3k8jM4K4kgokUem7m5I9AE7Bf2pK8qy1Ow0DqxvspyRD0LF3SWckWnk0F+JQv
+Kqd8mBc6dybCFEPfHNPRnsZc4jEQTFRSqkodGmnZdH05lexqsSEHfUW6gDLh8zpS
+LkzC6F+UdWBKLy7AfCavyACcvvMJYMC8GCOYY/zh/t+T0e++Q4PaOJr8/TA72eo1
+SznmZddFd6IdNk6ofqUOfdfJj10vJC0vPIaw4dHzXKNwFb0IEh+44p1/vjeZRe9q
+GhlklM9FvMRSyBTqQp/S9h2ZYCglTiljb1/NOScN2t41O0ST5v2ao+QsxD1/0Yyc
+ohBdB6/fkgpJum3w4gVAzTNVWMIvr2SKTUF6PPKNua+qFhc3bR5Y8q/REeSYan89
+0usDUEMXCNFU2b1sM29l3AN/QOCgRf+28Q7DB1wNGK+BCO8CSoTdox6WxMlC4/Xp
+HmsXj0vZPkbTXBRSyDCoaXvNhkNos0ycNRYZLQBtIUgqoQvae9XibEcgF4e/mr7M
+mrDXt6CAn1hx/xDoG4iA5/IQB71FeMr5AqTGvf0928LNYk7KXUlH0E/Wk8OZXwID
+AQABoAAwDQYJKoZIhvcNAQELBQADggIBAH5ypI8iurDm+Pq32+d+uZjHHJxUEC2D
+jKh5KpGlLNLr1dZe2CnDTVBUwz8NryPZSl6h6FsG1tFJwshqS+q9RNWKmPQSRp3A
+xgFAGgX4BL5oLil4dZVeylm5CN+6dH2SKlTpMgkQTOzitXCQH9239j96lxYj2PCA
+9thFnUetvXgJQ0qiYwBXkOOllwZCGcQchUy8U/LWA3ctegN205IpsNyJqGnDGhId
+s8M3Zed0/ebqMwsZPqzeKgcW3gm18ftWED7dwH6Yqjd+aJjNJY/8BYnMnZy7IfGg
+TkclJ8NJD6RSFERuak28+VgUZNPOPtHcuKSCL3e17Ok1wdOSyPzoD7bexA0xzEP6
+/t1hZFZZ7j1jM2jva3uGxPgWPCRRKPqxsBP4fTenuros9yMi8xSXhqobYnoqoezi
+xbOFkNkuWTEBVY0TKNfvdFQGSQHN1Ekm5Bcgra9GoyZKlIgKrpK+xu0/VxHq18tA
+Ghgp9+HW8921I5QwK8DVRqPpGb/eP+mGh5C2B5JMQXlaT7BGNTdlmsrKowbNnYeQ
+CJHBrtA3rlwkkffNw7pGgFSKMstIStwCR1zrDqjMU+T8F0Xo/JBQX6r9/b7JRVlJ
+Hu3V5xTXxq0sDQnskW/8e42WcjrZhqnK/wx3rLK2stlZTGCTt/ZPzHweX/2cndz/
+9VpeEsUKnnvH
+-----END CERTIFICATE REQUEST-----
```

Comment:
-----BEGIN CERTIFICATE REQUEST-----
MIIEVDCCAjwCAQAwDzENMAsGA1UEAwwEbmV3czCCAiIwDQYJKoZIhvcNAQEBBQAD
ggIPADCCAgoCggIBAONu7I+JYH/8/oCE/QaCpldz20SpnwJPJ8wmKzargYMITif2
u/NF5bCRQzrSrziIO73PyQIpN+7G3sUtzend0aMIGcgwP7+ShoLDqIK1adhVU2TY
2ij2DqRJK0NuUGgPgXgu30tn2ZdItGYnFxDjw4tSRKOWln+80zXcMk7LBAlN34hM
ubTwa+Vw3LG6HrAbLBSiMVCfHOEAseSFzoalSV+qP0qrtel/ByFrP5Oup5d7hB3g
88iWTxqCgzjI38oahj+dPADH+7EL0LKYV0844x4TJOfxqh2KbbsiX4A+nzMovPwA
K2w7CZgd6CeC3XlvnhrKGSauXEABsNNUZBlh6BORb60ReFEeUMti7+BcSoBKXAlO
MRjaJ3FE7riZ5iZFHdN0zSVcs2GKsIELTbGlumwXPnBuEK53gmlm6lIIoYKOG+OW
vWEEsyCSc2stHD3iRyi1iciqHzipnR9WLJpdKuXT/gmhZzwmKwoYPb07DbsJFwFP
L6dkmJ950c0dAvPEVRRXl+Ho0b8w4WsiVGuDddjcWqENM6UyzKoOxwJdTW3wJXTA
kBFC9M8VVghXv9/1zJrViIItwxRAM3IkDezwuz1k1620ktHjpQ0KSvCtvFHX5laf
OiIgvkIqlAPI9yAVpzu9ZvsJVtdDEmZW47ZBECVqNufO15s6YtlWHdyVIowvAgMB
AAGgADANBgkqhkiG9w0BAQsFAAOCAgEAW23se48KsUO1bYmovLyQwak8PnmZEbVj
ubBIBrutJRjt5KYJ2YL5VouhvFZtcXAdBHTljzXhsxyRSzyy6M8pgePJ62Ut/OVs
efMdWCAk7WJ+E1p+bibzd8qSByQ7VRHrbjxUlnpO/8jFXnY4WZkkrCKbb5unjzjr
Vi2c83PI4YIIMvi0BRSgpH4uI/qtg5P+3AKTELpMTiekUBf4msDibMosfDjzyTva
MAlIscxuQXjjC+Td/ZUITVdlTfE6/0Qu266ooJ/SKArwMxDpHF9Ptul2/6Yew1Sl
KqHU0bAUWSoTZR/GLYg69Xl1sy0i2H0gkxzW+MzFRN0BXFM0+tMc7Od6EoNyOJ7i
zrHeRhu8Mq/0XfMq2stUBFAgQCfiGf/1q+slX2M7YZZi5UymlVZpljQMszdL1Jqy
iYpudcmzFeSrRFZfGj40X83Zwxz3dd3q5Jru+D36uscv6DX0iNhtmp9nCYVlOaxw
tVpIH90TbY+KeFA8qhGPqPI0eZw1lW2hgTOaclOraygM9P5r95+jDsHWLmuenSJG
20qgqWED0onS9RqMcwpekvaUa/IcBN1MyQMPBxP0u/ROPZAUT4vIktRGA2gu9YLz
U3sLe00337OzgdolapNetcA6ZdObU29vmNbYpqOVn7KUEyqmgOtMCH4B+8XnEDMJ
EKIA8mRN54w=
-----END CERTIFICATE REQUEST-----

Reply:
If you want to change the CSR then please update the PR accordingly. If you need help with this just raise your questions and we are happy to help. :)